bin/n

```diff
@@ -1474,6 +1479,13 @@ function show_diagnostics() {
     echo_red "Neither curl nor wget found. Need one of them for downloads."
   fi
 
+  printf "\njq\n"
+  if command -v jq &> /dev/null; then
+    command -v jq && jq --version
+  else
+    echo "jq not found, can be used to get package.json values faster than node"
```

Comment:
```suggestion
    echo "jq not found. Optional install used for reading engine version from package.json."
```

Reply:
Yeah, I don't mind the phrasing -- I forgot to say that you should make it whatever fits...